docs/template_plugin/tests/functional/op_reference/reduction.hpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <vector>
+
+#include "base_reference_test.hpp"
+#include "ngraph_functions/builders.hpp"
+
+namespace reference_tests {
+namespace ReductionOpsRefTestDefinitions {
+
+struct ReductionParams {
+    template <class ET>
+    ReductionParams(const ngraph::PartialShape& inShape, const ngraph::element::Type& inType, const std::vector<ET> inValues,
+                    const std::vector<int64_t> axesValues, const std::vector<ET> outValues, const bool keepDims,
+                    const ngraph::helpers::ReductionType& reductType)
+            : dataShape(inShape), elemType(inType), inputData(CreateBlob(inType, inValues)), axesData(axesValues), refData(CreateBlob(inType, outValues)),
+              keepDimensions(keepDims), reductionType(reductType) {}
+    ngraph::PartialShape dataShape;
+    ngraph::element::Type elemType;
+    InferenceEngine::Blob::Ptr inputData;
```

Comment:
We have a Tensor struct which joins these structures.

Reply:
Good point. I have changed it to use Tensor for data input and output. As for axes I left vector as I want it to be part of the test case name